pkgs/development/libraries/libnatspec/default.nix

```
@@ -13,12 +13,12 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ popt ];
 
+  propagatedBuildInputs = lib.optional (!stdenv.isLinux) libiconv;
```

```suggestion
  propagatedBuildInputs = [ libiconv ];
```
libiconv is nil on linux.